package.json

```diff
@@ -81,8 +81,11 @@
     "@rollup/rollup-linux-x64-gnu": "^4.17.2"
   },
   "lint-staged": {
-    "*.js": "npm run lint:js",
-    "*.css": "npm run lint:css",
-    "**/*": "npm run lint:ls && npm run format"
+    "*.js": [
+      "npm run lint:js --",
+      "npm run format --"
+    ],
+    "*.css": "npm run lint:css --",
+    "*.*": "npm run lint:ls --"
```

Comment:
just making sure i understand, but this change looks like it would _not_ be running Prettier anymore on all files, just JS files?  Is there any way we could keep that previous behavior of formatting all files?

I think this should preserve the existing intentions while also accounting for the best practices you've applied here? 
```json
"lint-staged": {
  "*.js": "npm run lint:js --",
  "*.css": "npm run lint:css --",
  "*.*": [
    "npm run lint:ls --",
    "npm run format --"
  ]
}
```

---

Related, but I think I may have made a mistake in the original Stylelint configuration, in that it looks like we are still honoring stylistic rules with [the plugin](https://github.com/prettier/stylelint-prettier) I installed (was not my intention), which we shouldn't be doing since that's the whole point with Prettier (seem obvious in hindsight 😅 ).

This also causes tension in our lint staged as it commits the faux pax of having multiple tools trying to enforce / report style rules.  (I had setup ESLint to ignore style rules and that looks to be behaving as expected)?
<details>
  <pre>
    ➜  www.greenwoodjs.dev git:(issue-51-resolve-lint-staged) ✗ git commit -m "fix styles"
    ✔ Preparing lint-staged...
    ⚠ Running tasks for staged files...
      ❯ package.json — 2 files
        ↓ *.js — no files
        ❯ *.css — 1 file
          ✖ npm run lint:css -- [FAILED]
        ❯ *.* — 2 files
          ✔ npm run lint:ls --
          ✖ npm run format -- [KILLED]
    ↓ Skipped because of errors from tasks.
    ✔ Reverting to original state because of errors...
    ✔ Cleaning up temporary files...
    
    ✖ npm run lint:css --:
    
    src/styles/theme.css
      23:7  ✖  Replace "·" with "⏎"  prettier/prettier
    
    ✖ 1 problem (1 error, 0 warnings)
    
    
    > www.greenwoodjs.dev@0.0.1 lint:css
    > stylelint "./src/**/*.css" /Users/owenbuckley/Workspace/project-evergreen/www.greenwoodjs.dev/src/styles/theme.css
    
    
    ✖ npm run format --:
    
    > www.greenwoodjs.dev@0.0.1 format
    > prettier . --write /Users/owenbuckley/Workspace/project-evergreen/www.greenwoodjs.dev/package.json /Users/owenbuckley/Workspace/project-evergreen/www.greenwoodjs.dev/src/styles/theme.css
  </pre>
</details>

I think what I _should_ have done is install [**stylelint-prettier-recommended**](https://github.com/stylelint/stylelint-config-recommended) which now also benefits from actually catching some lint errors lol 🙃 
<details>
  <pre>
    ➜  www.greenwoodjs.dev git:(issue-51-resolve-lint-staged) ✗ npm run lint:css                     
    
    > www.greenwoodjs.dev@0.0.1 lint:css
    > stylelint "./src/**/*.css"
    
    
    src/components/capabilities/capabilities.css
      44:1  ✖  Expected selector ".active strong" to come before selector ".section:hover strong"  no-descending-specificity
      70:1  ✖  Unexpected duplicate selector ".heading", first used at line 12                     no-duplicate-selectors
      88:1  ✖  Unexpected duplicate selector ".sections", first used at line 21                    no-duplicate-selectors
    
    src/components/copy-to-clipboard/copy-to-clipboard.css
      7:3  ✖  Unexpected duplicate "background-color"  declaration-block-no-duplicate-properties
    
    src/components/footer/footer.module.css
      14:3  ✖  Unexpected shorthand "flex" after "flex-basis"  declaration-block-no-shorthand-property-overrides
    
    src/components/hero-banner/hero-banner.module.css
      90:3  ✖  Unexpected duplicate "display"         declaration-block-no-duplicate-properties
      94:3  ✖  Unexpected duplicate "vertical-align"  declaration-block-no-duplicate-properties
    
    src/components/side-nav/side-nav.module.css
      69:3  ✖  Expected selector ".compactMenuSectionHeading a" to come before selector ".compactMenu   no-descending-specificity
               a:hover"
      74:1  ✖  Unexpected duplicate selector ".compactMenuSectionHeading", first used at line 66        no-duplicate-selectors
      95:3  ✖  Expected selector ".compactMenuSectionListItemActive a" to come before selector          no-descending-specificity
               ".compactMenu a:hover"
    
    src/components/header/header.module.css
      111:1  ✖  Expected selector ".mobileMenuListItem a" to come before selector ".navBarMenuItem      no-descending-specificity
                a:hover"
    
    ✖ 11 problems (11 errors, 0 warnings)
      3 errors potentially fixable with the "--fix" option.
  </pre>
</details>

----

So with all that said, I think we're pretty close to get this setup properly
1. [ ] Swap out **stylelint-prettier/recommended** (config and dependency) with **stylelint-config-recommended** in _stylelint.config.js_
1. [ ] Fix up any new CSS linting checks / sanity test UI
1. [ ] tweak link staged tasks to reflect original intent + best practices
1. [ ] (bonus / optional / whatever lol) put lintstaged config in its own config file